### PR TITLE
modified HP 5406 and 5412

### PR DESCRIPTION
**Pull request by francesco-noc**
I found that Aruba 5406[0] is different from HP 5406. They have different part number.

[0] https://support.hpe.com/hpesc/public/docDisplay?docId=emr_na-c04334067#Ref5

### CODE REVIEW

**Comment by danner26** [comment]
Hello, today we finished migrating all of the files and slugs to their new proper naming convention as described in the README. It looks like this PR has a merge conflict. I do not have access to modify this branch, so please resolve it by using the slug/model name defined in the master branch and then push it to this PR.

**Comment by francesco-noc** [comment]
Sorry, they all have module bays.

Regarding the models.. it was another enjoying move by HP. There was the HP 5406R zl2 (J9850A) then when HP acquired Aruba, the same switch was rebranded with Aruba logo on top and became Aruba 5406R zl2 with part number J9821A

File: device-types/HPE/HP-5412R-zl2.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
+---
+manufacturer: HPE
+model: HP 5412R-zl2
```
[the remark this answers is not in the document]
Comment (danner26) [reply device-types/HPE/HP-5412R-zl2.yaml:3]: @francesco-noc can you confirm if this is the `Aruba 5412R ZL2`?

[the remark this answers is not in the document]
Comment (francesco-noc) [reply device-types/HPE/HP-5412R-zl2.yaml]: As for 5406. There were HP 5412R zl2 and then Aruba 5412R zl2

**Comment by danner26** [comment]
All looks good to me then, anything else you need before we merge?

**Review by danner26 — APPROVED** [review]

**Pull request merged**